plugins/Model_GAN/Model.py

```diff
@@ -56,6 +59,11 @@ def one_minus(x): return 1 - x
         # img as input => generates encoded represenation and reconstructed image => determines validity 
         self.adversarial_autoencoderA = Model(img, [reconstructed_imgA, out_discriminatorA])
         self.adversarial_autoencoderB = Model(img, [reconstructed_imgB, out_discriminatorB])
+        
+        if self.gpus > 1:
+            self.autoencoder_A = multi_gpu_model( self.adversarial_autoencoderA , self.gpus)

```

Comment:
shouldn't it be:
`self.self.adversarial_autoencoderA = multi_gpu_model( self.adversarial_autoencoderA , self.gpus)`?

Reply:
Oops, good catch.  Yes.  My testing was minimal on GAN since my GPU only has 4gb of ram and I get OOMs on that one.  I was borrowing another GPU and only ran enough of a test to verify that the model file is loaded and saved correctly.  I've fixed and committed.